src/chrome/content/scaffold/aceWrapper.js

```diff
@@ -37,6 +37,69 @@ window.addEventListener("DOMContentLoaded", function(e) {
 	JavaScriptMode = require("ace/mode/javascript").Mode;
 	TextMode = require("ace/mode/text").Mode;
 	EditSession = require("ace/edit_session").EditSession;
+
+	ace.config.loadModule('ace/ext/language_tools', function () {
+		var snippetManager = ace.require("ace/snippets").snippetManager;
+		snippets = [{
+			content: "for (let i=0; i<${1:arrayName}.length; i++) {\n\t${2:// continue here}\n}\n",
+			name: "simple for loop"
```

Comment:
If we're going to bother with stuff like this, probably better to default to `for...of`, with two placeholders.